### PR TITLE
Add support for PolarDB/Ganos

### PR DESCRIPTION
**Pull request by erchen-cz**
Please see 3dctiydb-docs [#40](https://github.com/3dcitydb/3dcitydb-docs/pull/40) for more details

### CODE REVIEW

**Review by yaozhihang — COMMENTED** [review]
Thanks very much @erchen-cz for the PR. I have two comments after my review. It would also be nicer, if you could provide us a PolarDB/Ganos database instance for testing purposes.

File: postgresql/SQLScripts/CREATE_DB.sql [hunk 1]
```diff
@@ -32,16 +32,12 @@ SET client_min_messages TO WARNING;
...
---// check if the PostGIS raster extension is available
-SELECT EXISTS(SELECT 1 AS create_raster FROM pg_extension WHERE extname = 'postgis_raster') AS postgis_raster_exists;
-\gset
-
---// break if the PostGIS version >= 3 and the PostGIS raster extension is not installed
-SELECT CASE WHEN :'postgis_version' < '3' OR :'postgis_raster_exists' = 't'
+--// break if the raster data type is not available
+SELECT CASE WHEN :'raster_type_exists' = 't'
   THEN 'UTIL/HINTS/DO_NOTHING.sql'
   ELSE 'UTIL/HINTS/HINT_ON_MISSING_RASTER_EXTENSION.sql'
```
[the remark this answers is not in the document]
Comment (erchen-cz) [reply postgresql/SQLScripts/CREATE_DB.sql:42]: It looks better and I will fix this

File: README.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ System requirements
 -------------------
 * PostgreSQL DBMS >= 9.6 with PostGIS extension >= 2.3
 * Oracle DBMS >= 10g R2 with Spatial or Locator option
+* PolarDB for PostgresSQL >= 1.1.22 with Ganos extension >= 4.6
```
[the remark this answers is not in the document]
Comment (erchen-cz) [reply README.md:61]: Currently PolarDB for PostgreSQL is 1.1.x while Ganos with different version. Ganos 4.6 is included in 1.1.22, in which 3DCityDB is tested.  If Ganos version is pointed out here, patch version of PolarDB can be ignored.

**Review by yaozhihang — APPROVED** [review]

**Comment by erchen-cz** [comment]
Can anyone help to merge this PR?

**Comment by erchen-cz** [comment]
> @erchen-cz, thanks for your PR. It indeed seems easy to include your requested changes now. If we make PolarDB/Ganos an officially supported database system for the 3DCityDB, we also need to make sure that future changes and releases of the 3DCityDB are continously tested against PolarDB/Ganos or that issues and change requests specific to PolarDB/Ganos are worked on and answered. What do you think about this?

Hi @clausnagel , we have offered a PolarDB/Ganos instance in Germany for 3DCityDB continously testing. The database will be reserved for a long time and be updated if newer PolarDB/Ganos is released.  Also we can help to test the new 3DCityDB version against PolarDB/Ganos  and make modification if there is any problem.

If you want  more information about the instance,  I think @yaozhihang would like to help.

**Comment by clausnagel** [comment]
> Also we can help to test the new 3DCityDB version against PolarDB/Ganos and make modification if there is any problem.

Great! How about adding someone from your PolarDB/Ganos team as contributor to the 3DCityDB GitHub team? This way we can directly assign issues or tasks via the GitHub platform.

**Comment by erchen-cz** [comment]
> someone fro

The issues or tasks related to PolarDB/Ganos can be assigned to me first, and then I can make other guys involved.   Please feel free to make me as issue owner if necessary.

**Pull request merged**